### PR TITLE
Add new link

### PR DESCRIPTION
**Pull request by proyb6**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -251,6 +251,7 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
  * [realtime-todo-app](https://github.com/Angarsk8/realtime-todo-app) - Realtime Todo application developed with Kemal, Websockets, React, ES2015 and PostgreSQL
  * [rocky](https://github.com/codingphasedotcom/rocky) - React Over Crystal Kemal and Yarn
  * [try.cr](https://github.com/maiha/try.cr) - Try monad
+ * https://github.com/KieranP/Game-Of-Life-Implementations - Compare Game of Life in Crystal and other languages
```
Comment (veelenga) [inline README.md]: We need to follow the existed way to add the links. See the examples above.

File: README.md [hunk 2]
```diff
@@ -251,6 +251,7 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
  * [realtime-todo-app](https://github.com/Angarsk8/realtime-todo-app) - Realtime Todo application developed with Kemal, Websockets, React, ES2015 and PostgreSQL
  * [rocky](https://github.com/codingphasedotcom/rocky) - React Over Crystal Kemal and Yarn
  * [try.cr](https://github.com/maiha/try.cr) - Try monad
+ * [game-of-life-implementations] (https://github.com/KieranP/Game-Of-Life-Implementations) - Game of Life in Crystal and other languages
```
Comment (hugoabonizio) [inline README.md]:
```suggestion
 * [game-of-life-implementations](https://github.com/KieranP/Game-Of-Life-Implementations) - Game of Life in Crystal and other languages
```

**Comment by proyb6** [comment]
Thank you for the correction, I forgot about it.

**Comment by veelenga** [comment]
@proyb6 it is still not corrected, you need to commit it.

**Pull request closed without merging**